pkgs/by-name/sw/swig/package.nix

```diff
@@ -21,11 +21,13 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   PCRE_CONFIG = "${pcre2.dev}/bin/pcre-config";
```

Comment:
There's already this, either fix using it or do what ive done before https://github.com/NixOS/nixpkgs/blob/53029fdf6ae06baff5d0bf627b5a9b9825cc3126/pkgs/by-name/cp/cppcheck/package.nix#L63

Reply:
`PCRE_CONFIG` does not seem to do anything. It was introduced in https://github.com/NixOS/nixpkgs/commit/f8d83938a757b26802535af14ab1281d369f0fe4#diff-a2ce329463359c12519f5692e51fbc8892ff923c4a646aa865890784a233df52R14.

I propose to remove it.